locales/fr.yaml

```diff
@@ -0,0 +1,553 @@
+cancel: Annuler
+pay_invoice: Merci de payer cette facture de ${amount} sats à ${fiatAmount} ${currency} pour démarrer la transaction.
+payment_received: |
+  🤑 Paiement reçu!
+
+  Maintenant j'ai besoin que l'acheteur envoie une facture pour que j'envoie les satoshis vers son portefeuille. Je vous informerai tous les deux dès que l'acheteur envoie la facture
+
+  Un peu de patience. Le collatéral vous sera recrédité si l'acheteur ne se montre pas coopératif.
+someone_took_your_order: |
+  🤖 Quelqu'un a récupéré ton offre et m'a déjà envoyé les sats, merci d'utiliser un processeur de paiement fiat qui te permet d'envoyer le paiement instantanément et qui ne permet pas de geler les fonds.
+
+  Si pour une raison quelconque le processeur de paiement met le paiement en pause et que les fonds ne sont pas arrivés en moins de ${expirationTime}, les sats seront retournés au vendeur, mettant l'acheteur dans une position risquée. Je ne peux pas obliger le vendeur à vendre ses sats de nouveau.
+
+  Seller Reputation: ${rate}
+
+  Si tu es d'accord avec ce qui précède, clique sur le bouton pour continuer 👇
+you_took_someone_order: |
+  🤖 Tu as pris cet offre de vente, merci d'utiliser un processeur de paiement fiat qui te permet d'envoyer le paiement instantanément et qui ne permet pas de geler les fonds\\.
+
+  *Si pour une raison quelconque le processeur de paiement met le paiement en pause et que les fonds ne sont pas arrivés en moins de ${expirationTime}, les sats seront retournés au vendeur, mettant l'acheteur dans une position risquée. Je ne peux pas obliger le vendeur à vendre ses sats de nouveau\\.*
+
+  Si tu es d'accord avec ce qui précède, clique sur le bouton pour continuer 👇
+get_in_touch_with_seller: |
+  🤖 ID de l'offre : #${orderId}
+
+  Prends contact avec le vendeur, dont le nom d'utilisateur est @${sellerUsername}, pour obtenir les détails pour envoyer l'argent ; tu dois envoyer ${fiatAmount} ${currency} en utilisant ${paymentMethod}.
+
+  Dès que tu envoies l'argent, informe-moi avec cette commande 👇
+fiatsent_order_cmd: /fiatsent
+get_in_touch_with_buyer: |
+  🤖 ID d'offre : #${orderId}
+
+  Prends attache avec @${buyerUsername} pour lui fournir les informations afin de t'envoyer ${fiatAmount} ${currency} via ${paymentMethod}. Ne débloque pas les sats sans t'assurer que l'acheteur t'a envoyé le paiement fiat.
+
+  Dès que tu as vérifié que le paiement est arrivé, tu dois débloquer les sats.
+buyer_took_your_order: |
+  🤖 ID d'offre : #${orderId}
+
+  @${buyerUsername} a récupéré ton offre et veut acheter tes sats. Contacte-le pour lui indiquer comment t'envoyer ${fiatAmount} ${currency} via ${paymentMethod}.
+
+  Dès que tu as vérifié que le paiement est arrivé en totalité, tu dois débloquer les sats
+waiting_seller_to_pay: 'J''ai envoyé une requête de paiement au vendeur pour qu''il envoie tes sats concernant l''offre dont la référence est #${orderId}, dès que le paiement est fait je vous informerai tous les deux.'
```

Comment:
When you need to use single quote (`'`) like this, the easiesy way to do it is using double quotes (`"`), example

`waiting_seller_to_pay: :J'ai envoyé une requête de paiement au vendeur pour qu'il envoie tes sats concernant l'offre dont la référence est #${orderId}, dès que le paiement est fait je vous informerai tous les deux."`